### PR TITLE
make examples copile

### PR DESCRIPTION
**Pull request by bakito**
The examples as they are checked in are not compiling.

This Pull request fixes compilation errors in the examples and moves all examples into separate folders.

### CODE REVIEW

**Comment by c-seeger** [comment]
hi @bakito thanks for your contribution.

File: .travis.yml [hunk 1]
```diff
@@ -24,12 +24,12 @@ before_install:
 
 script: 
   - golint `go list`
-  - golint ./examples/*.go
+  - golint ./examples/*/*.go
```
Comment (c-seeger) [inline .travis.yml:27]: Is this path change working?

**Comment by c-seeger** [comment]
the rest LGTM

**Comment by bakito** [comment]
yes the travis build was running https://travis-ci.com/github/bakito/confluence-go-api/builds/182208941

**Pull request merged**